.ci/appveyor.bat

```diff
@@ -0,0 +1,56 @@
+@echo off
+
+::----------------------------------------------------------------------
+:: Appveyor bat file to run on local for windows users.
+:: Example of usage
+:: ./.ci/appveyor.bat  verify_without_checkstyle
+::----------------------------------------------------------------------
+
+SET OPTION=%1
+goto :%OPTION% 2>nul
+IF ERRORLEVEL 1 CALL :DEFAULT_CASE
+ECHO Done.
+EXIT
+
+:sevntu
+mvn -e verify -DskipTests -DskipITs -Dpmd.skip=true^
+ -Dspotbugs.skip=true -Djacoco.skip=true -Dxml.skip=true
+    goto END_CASE
+
+:verify_without_checkstyle
+mvn -e verify -Dcheckstyle.ant.skip=true -Dcheckstyle.skip=true
+  goto END_CASE
+
+:verify_without_checkstyle_JDK11
+mvn -e verify -Dcheckstyle.ant.skip=true -Dcheckstyle.skip=true -Dspotbugs.skip=true
+  goto END_CASE
+
+:verify_without_checkstyle_JDK12
+mvn -e verify -Dcheckstyle.ant.skip=true -Dcheckstyle.skip=true -Dspotbugs.skip=true^

```

Comment:
Please add comment similar to travis.sh.
> `# powermock doesn't support modifying final fields in JDK12`

Reply:
Done